### PR TITLE
Support allow_other/allow_root Fuse mount options (#7900)

### PR DESCRIPTION
**Pull request by calvinjia**
cherry-pick of Alluxio/alluxio#7900
cherry-pick commit is Alluxio/alluxio@4c22e33b8a61

### CODE REVIEW

File: integration/fuse/bin/alluxio-fuse [hunk 1]
```diff
@@ -114,18 +120,31 @@ fi
...
-    if [[ $# -eq 2 ]]; then
-      mount_fuse $2 /
+    shift
+    while getopts "o:" option; do
+      case "${option}" in
+        o)
+          mount_option=${OPTARG}
+          ;;
+        *)
+          echo -e "${USAGE}" >&2
```
Comment (liuhongtong) [inline integration/fuse/bin/alluxio-fuse]: should use USAGE_MSG?

Comment (apc999) [reply integration/fuse/bin/alluxio-fuse]: it should be `USASGE_MSG` defined https://github.com/Alluxio/alluxio/pull/9497/files#diff-353dcb496e9c07a6d894ece8ecb85c8dR107 @calvinjia

**Comment by calvinjia** [comment]
I think this PR is a subset of #9487 ? @gpang

File: integration/fuse/bin/alluxio-fuse (hunk 1 above)
Comment (calvinjia) [reply integration/fuse/bin/alluxio-fuse]: ack

**Comment by calvinjia** [comment]
Jenkins, test this please.

**Comment by calvinjia** [comment]
alluxio-bot, merge this please.

**Pull request merged**